### PR TITLE
add Qiime package

### PR DESCRIPTION
**Pull request by bgruening**
This add the Qiime package to get a new community effort from @lparsons rolling!

For more information see here: http://dev.list.galaxyproject.org/QIIME-tools-for-Galaxy-WIP-call-for-collaborators-td4668035.html

### CODE REVIEW

**Comment by bgruening** [comment]
@lparsons do you mind testing this from the TTS?

File: packages/package_python_2_7_qiime_1_9_1/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                <action type="set_environment">
+                    <environment_variable action="append_to" name="PYTHONPATH">$INSTALL_DIR</environment_variable>
+                    <environment_variable action="append_to" name="PYTHONPATH">$ENV[PYTHONPATH_NUMPY]</environment_variable>
+                    <environment_variable action="append_to" name="PYTHONPATH">$ENV[PYTHONPATH_MATPLOTLIB]</environment_variable>
+                    <environment_variable action="append_to" name="PYTHONPATH">$ENV[PYTHONPATH_PANDAS]</environment_variable>
+                    <environment_variable action="append_to" name="PYTHONPATH">$ENV[PYTHONPATH_SCIPY]</environment_variable>
+                    <environment_variable action="append_to" name="PYTHONPATH">$ENV[PYTHONPATH_REQUESTS]</environment_variable>
+                    <environment_variable action="prepend_to" name="PATH">$ENV[PATH_NUMPY]</environment_variable>
+                    <environment_variable action="set_to" name="PYTHONPATH_QIME">$INSTALL_DIR</environment_variable>
+                    <environment_variable action="set_to" name="QIME_ROOT_PATH">$INSTALL_DIR</environment_variable>
```
Comment (erasche) [inline packages/package_python_2_7_qiime_1_9_1/tool_dependencies.xml]: QIIME?

File: packages/package_python_2_7_qiime_1_9_1/tool_dependencies.xml [hunk 2]
```diff
@@ -0,0 +1,97 @@
+<?xml version='1.0' encoding='utf-8'?>
+<tool_dependency>
+    <package name="numpy" version="1.9">
+        <repository name="package_python_2_7_numpy_1_9" owner="iuc" prior_installation_required="True" />
+    </package>
+    <package name="python" version="2.7.10">
+        <repository name="package_python_2_7_10" owner="iuc" prior_installation_required="True" />
```
Comment (yhoogstrate) [inline packages/package_python_2_7_qiime_1_9_1/tool_dependencies.xml]: All the other packages that depend on python depend on "package_python_2_7". Is there a particular reason to switch to "package_python_2_7_10" ?

Comment (bgruening) [inline packages/package_python_2_7_qiime_1_9_1/tool_dependencies.xml]: Which ones? But yes. I introduced `package_python_2_7` one year ago and thought there will be no major API problems. It turns out I was wrong. So I will have `package_python_2_7_10` which has a latest version of setuptools, the new ssl library and so on. 
From now on I will only use this package or any higher number.

File: packages/package_python_2_7_qiime_1_9_1/tool_dependencies.xml (hunk 1 above)
Comment (bgruening) [inline packages/package_python_2_7_qiime_1_9_1/tool_dependencies.xml]: Nice catch. Actually with the latest Galaxy patch which you merged this is not necessary anymore. I will leave it in for the time being.

**Comment by bgruening** [comment]
Anyone?

**Comment by erasche** [comment]
Checking URLs with new planemo stuff (and fixing bugs in that), will get this merged shortly.

**Comment by erasche** [comment]
:+1:

**Pull request merged**